### PR TITLE
Add community configuration  page

### PR DESCRIPTION
**Pull request by sladyn98**
This page displays all of the community configuration pages. For now dummy data has been inserted which will later be replaced by the github api. Do let me know if the design looks good and I can integrate the api
Let us get this initial design merged in so that we can build on top of this 

Screenshot from 2020-07-13 14-23-22

### CODE REVIEW

**Comment by sladyn98** [comment]
@oleg-nenashev Makes perfect sense, I did it this way because the team thought it would be better to have it all in a single repository. So I guess I can create a different repository on my personal account and then host it on jenkins later ? 
CC @kwhetstone @baymac

**Comment by kwhetstone** [comment]
I think for this project keeping the templates in the repository is ok. , Especially since right now, the focus is on running things locally.  If you had to spin this up locally, you'd probably want to pull in the templates and fill in pieces of information before generating the war file.

File: frontend/src/components/CommunityConfiguration/communityConfigLayout.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import PluginCard from '../PluginCards/pluginCard'
+import { Container, Row, Col, Button} from 'reactstrap';
+import { Pagination, PaginationItem, PaginationLink, Spinner } from 'reactstrap';
+import ConfigurationCard from './configurationCard';
+
+class communityConfigLayout extends React.Component {
+
+    state = {
+        search:"",
+        data: [
```
Comment (kwhetstone) [inline frontend/src/components/CommunityConfiguration/communityConfigLayout.js:11]: Is there a way to dynamically load these?  That would make it easier to work with here.  At the same time, I wonder if there's a way to go through and add tags for these pieces for search.  That could be another ticket.

Comment (sladyn98) [reply frontend/src/components/CommunityConfiguration/communityConfigLayout.js:11]: Yeah working on getting these to load dynamically

File: frontend/src/components/CommunityConfiguration/communityConfigLayout.js [hunk 2]
```diff
@@ -8,43 +8,16 @@ class communityConfigLayout extends React.Component {
...
-                "html": "https://github.com/jenkins-zh/jenkins-formulas/blob/master/formulas/k8s.yaml"
-            }
-        }
-        ],
+        data: [],
         isLoading: true
     }
 
+    async componentDidMount() {
+        const response = await fetch('https://api.github.com/repos/jenkins-zh/jenkins-formulas/contents/formulas');
```
Comment (sladyn98) [inline frontend/src/components/CommunityConfiguration/communityConfigLayout.js:16]: We need to change this to our URL once we get the PR merged in

**Pull request merged**